apps/teams-test-app/src/components/MeetingAPIs.tsx

```diff
@@ -327,6 +327,52 @@ const SetOptions = (): React.ReactElement =>
     },
   });
 
+const RequestAppAudioHandling = (): React.ReactElement =>
+  ApiWithCheckboxInput({
+    name: 'requestAppAudioHandling',
+    title: 'App Handles the Audio channel',
+    onClick: async (input: boolean, setResult: (arg0: string) => void) => {
+      const callback = (
+        error: SdkError | null,
+        requestAppAudioHandlingSdkResponse: meeting.RequestAppAudioHandlingSdkResponse | null,
+      ): void => {
+        if (error) {
+          setResult(JSON.stringify(error));
+        } else {
+          setResult('requestAppAudioHandling() succeeded: ' + JSON.stringify(requestAppAudioHandlingSdkResponse));
+        }
+      };
+      const callbackMicMuteStateChangedHandler = (micState: meeting.MicState): void => {
+        if (!micState) {
+          throw new Error('micStatus should be MicState');
```

Comment:
This error message seems wrong now, should probably be "micState should not be null"